django/db/backends/oracle/schema.py

```diff
@@ -152,6 +153,11 @@ def _generate_temp_name(self, for_name):
         return self.normalize_name(for_name + "_" + suffix)
 
     def prepare_default(self, value):
+        if hasattr(value, 'as_sql'):
+            sql, params = self.prepare_param(value)
+            if isinstance(value, Value):
+                params = tuple(self.quote_value(p) for p in params)
+            return sql % params
```

Comment:
I believe `params` should no longer contain any more `Expression` objects, and I ran the test suite with this change and didn't see any related failures. Trying to special case `Value` (and other expressions like it) seems like a losing battle because they may be nested inside other expressions.
```suggestion
            return sql % tuple(self.quote_value(p) for p in params)
```

Reply:
Yes, that seems to work. Thanks